index.js

```diff
@@ -404,6 +404,18 @@ async function runScan({
       out.json = REPORT_FILE;
       break;
     }
+    case "cyclonedx": {
```

Comment:
Instead of `cyclonedx`, should this be `cyclonedx-xml`? and should the file extensions be `.cdx.xml` and `.cdx.json`?

Reply:
> Instead of cyclonedx, should this be cyclonedx-xml? 

I guess it should be compatible with [grype outputs](https://github.com/anchore/grype?tab=readme-ov-file#output-formats) but I can change to whatever you like.

> and should the file extensions be .cdx.xml and .cdx.json?

Good idea. Changed

Reply:
@ps-e grype supports `cyclonedx-xml` (and also `cyclonedx`, which also results in the XML version). I think we should use the XML extensions for clarity here; and update all the descriptions to be clear that this is XML format (just as you have for the JSON variants). Other than that this PR looks great!

Reply:
I went ahead and updated this, please let me know if that's a problem!